dotnet/src/VectorDataIntegrationTests/AzureAISearchIntegrationTests/AzureAISearchIntegrationTests.csproj

```diff
@@ -17,6 +17,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.NET.Test.Sdk"/>
+    <PackageReference Include="Azure.Identity"/>
```

Comment:
FWIW things seem to be working fine for me without this, any context?

Reply:
I was getting a compiler error when I was trying to use the client ctor that accepts azure token credentials.